index.js

```diff
@@ -76,6 +76,7 @@ smartcar.isExpired = function(expiration) {
  * @param {Number} [paging.limit] - number of vehicles to return
  * @param {Number} [paging.offset] - index to start vehicle list
  * @return {Promise.<module:smartcar~VehicleIds>} A promise with the vehicle ids.
+ * @throws {SmartcarError}
```

Comment:
I think we should be super explicit for all these `@throws` jsdoc comments.
```suggestion
 * @throws {SmartcarError} - an instance of SmartcarError. See the [errors section of the README](https://github.com/smartcar/node-sdk/tree/master/doc#errors) for all possible errors.
```

If we want to be extra spicy, we can even have multiple `@throws` in the jsdoc for methods that call for it (e.g. FuEV endpoints that can throw VehicleNotCapable and SmartcarNotCapable).

```
 * @throws {SmartcarError} - an instance of SmartcarError. See the [errors section of the README](https://github.com/smartcar/node-sdk/tree/master/doc#errors) for all possible errors.
 * @throws {VehicleNotCapableError} - description...
 * @throws {SmartcarNotCapableError} - description...
```